src/App.tsx

```diff
@@ -3,16 +3,17 @@ import QuizTemplate from './components/QuizTemplate';
 import { useState } from 'react';
 
 
-const App = () => {
+const App: React.FC = () => {
   const [isHomePage, setIsHomePage] = useState(true);
   const toggleHomePage = () => setIsHomePage(!isHomePage);
   return (
     <>
-      {isHomePage ?
-        <WelcomePage start={toggleHomePage} /> :
-        <QuizTemplate home={toggleHomePage} />
-      }
-    </>
+    {
+      isHomePage?
+        <WelcomePage start = { toggleHomePage } /> :
+  <QuizTemplate home={ toggleHomePage } />
+}
+</>
```

Comment:
This formatting looks a little off. Please fix it using something like prettier. 

Preferred format:
```
    <>
      {isHomePage ? (
        <WelcomePage start={toggleHomePage} />
      ) : (
        <QuizTemplate home={toggleHomePage} />
      )}
    </>
```